packages/core/src/dialects/sqlite/query-generator-typescript.ts

```diff
@@ -54,6 +54,15 @@ export class SqliteQueryGeneratorTypeScript extends MySqlQueryGenerator {
       throw new Error(`Providing a columnName in getForeignKeyQuery is not supported by ${this.dialect.name}.`);
     }
 
-    return `PRAGMA foreign_key_list(${this.quoteTable(tableName)})`;
+    const quotedTable = this.quoteTable(tableName);
```

Comment:
It shouldn't be quoted but escaped, pragma_foreign_key_list expects a string, and the value we select as tableName is a string too

Reply:
Done, added an `escapeTable` function for it since it doesn't seem we have something that worked for sqlite yet